Adyen/CheckoutConfiguration/ConfigurationBuilder.swift

```diff
@@ -0,0 +1,29 @@
+//
+// Copyright (c) 2025 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import Foundation
+
+@resultBuilder
+public struct ConfigurationBuilder {
```

Comment:
```suggestion
public struct CheckoutConfigurationBuilder {
```

Reply:
It's a public struct and `ConfigurationBuilder` is quite generic - so maybe `CheckoutConfigurationBuilder` is a safer name :)

Reply:
It was CheckoutConfigurationBuilder but thought to myself, why not ConfigurationBuilder, how common could that be